src/index.ts

```diff
@@ -78,7 +78,7 @@ function inlineResourcesFromString(content: string, urlResolver: UrlResolver) {
  * @return {string} The content with all templates inlined.
  */
 function inlineTemplate(content: string, urlResolver: UrlResolver) {
-  return content.replace(/templateUrl:\s*['"]([^'"]+?\.html)['"]/g, function (_m, templateUrl) {
+  return content.replace(/templateUrl:\s*[`'"]([^`'"]+?\.html)[`'"]/g, function (_m, templateUrl) {
```

Comment:
backtick support. We needed this because our coding linter will ding us if we don't use backticks for consistency.